contrib/objectstore/subscriber/marshaller/custom1/custom1.go

```diff
@@ -0,0 +1,42 @@
+package custom1
+
+import (
+	"github.com/skydive-project/skydive/flow"
+)
+
+// Flow represents a transformed flow
+type Flow struct {
+	UUID             string
+	LayersPath       string
+	Network          *flow.FlowLayer
+	Transport        *flow.TransportLayer
+	LastUpdateMetric *flow.FlowMetric
+	Metric           *flow.FlowMetric
+	Start            int64
+	Last             int64
+	FinishType       flow.FlowFinishType
+}
+
+// Marshaller is a custom marshaller for flows
+type Marshaller struct {
+}
+
+// Marshal transforms a flow before being stored
+func (m *Marshaller) Marshal(f *flow.Flow) interface{} {
```

Comment:
Marshaller/Marshal are probably not good names as it leads to think they are going to marshal while they don't in fact. What about FowTransformer ?